src/index.js

```diff
@@ -318,6 +318,28 @@ function cloneElement(element, props, ...children) {
 	return normalizeVNode(preactCloneElement(...cloneArgs));
 }
 
+function validatePropTypes (vnode) {
+	let componentClass = typeof vnode.nodeName === 'function' ? vnode.nodeName : vnode.type;
+
+	if (typeof componentClass !== 'function') return;
+	let name = (
+		componentClass.prototype.displayName
```

Comment:
The tests would fail. It was returning the function you use to construct the pfcs. This was before I was testing on 8. I'll adjust and try again.